tests/stests/workloads/create_workloads.robot

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 Elektrobit Automotive GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+*** Settings ***
+Documentation    Tests to verify that Ankaios can create Podman workloads.
+Resource     ../../resources/ankaios.resource
+Resource    ../../resources/variables.resource
+
+*** Test Cases ***
+# [stest->swdd~agent-supports-podman~2]
+# [stest->swdd~podman-create-workload-runs-workload~1]
+# [stest->swdd~podman-delete-workload-stops-and-removes-workload~1]
+Test Ankaios Podman create and remove workloads
+    [Setup]        Run Keywords    Setup Ankaios
+
+    # Preconditions
+    # This test assumes that all containers in the podman have been created with this test -> clean it up first
+    Run Process    podman    rm    -a    -f    timeout=20    shell=True
```

Comment:
Shouldn't we extract this cleanup procedure into an own precondition text with a certain mapping?  Because that the containers are not running is a precondition.

Reply:
Fixed. See:
https://github.com/eclipse-ankaios/ankaios/pull/54/commits/5d46ef07d18e53eaaf729575a0a035fb14f35a5e#diff-de8975bacb16894b4d4a4a7ab83aea48caa9c25597bbac565f094309a74380bc

Let me know if you have any comment to this.